src/app/static/src/tests/components/runWorkflow/runWorkflowRun.test.ts

```diff
@@ -1,64 +1,262 @@
 import {shallowMount} from "@vue/test-utils";
-import runWorkflowRun from "../../../js/components/runWorkflow/runWorkflowRun.vue"
+import runWorkflowRun from "../../../js/components/runWorkflow/runWorkflowRun.vue";
+import {mockAxios} from "../../mockAxios";
+import ErrorInfo from "../../../js/components/errorInfo.vue";
 
 describe(`runWorkflowRun`, () => {
+
+    const changelogTypes = ["internal", "public"]
+    const source = ["prod", "uat"]
+    const runMetadata = {
+        git_branches: [],
+        metadata: {
+            changelog_types: changelogTypes,
+            git_supported: false,
+            instances_supported: true,
+            instances: {
+                source: source,
+                annex: ["one"]
+            }
+        }
+    }
+
+    const workflowSummaryMetadata = [
+        {name: "interim report", date: "2021-05-19T16:28:24Z", email: "test@example.com", key: "fake"},
+        {name: "interim report2", date: "2021-06-19T16:28:24Z", email: "test@example.com2", key: "fake2"}
+    ]
+
+    beforeEach(() => {
+        mockAxios.reset();
+        mockAxios.onGet('http://app/report/run-metadata')
+            .reply(200, {"data": runMetadata});
+        mockAxios.onGet('http://app/workflows')
+            .reply(200, {"data": workflowSummaryMetadata});
+    })
+
     const getWrapper = () => {
-        return shallowMount(runWorkflowRun, {propsData: {workflowMetadata: {}}})
+        return shallowMount(runWorkflowRun,
+            {
+                propsData: {workflowMetadata: {}},
+                data() {
+                    return {
+                        selectedInstances: {},
+                        workflowName: "",
+                        runMetadata: null,
+                        changeLogTypeValue: "",
+                        changeLogMessage: "",
+                        workflows: [],
+                        workflowNameError: ""
+                    }
+                }
+            }
+        )
     }
 
     it(`it renders workflow run page correctly`, () => {
         const wrapper = getWrapper()
         expect(wrapper.find("#run-header").text()).toBe("Run workflow")
     })
 
-    it(`can render workflow-name elements`, () => {
+    it(`can render workflow-name elements`, async (done) => {
```

Comment:
Don't need the top-level `async` on tests which include `setTimeout` unless they use `await` at the top level, since you've declared `setTimeout`'s method as `async`